ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Decision/DeclineReason.cshtml.cs

```diff
@@ -16,15 +16,15 @@ namespace ApplyToBecomeInternal.Pages.TaskList.Decision
 	public class DeclineReasonModel : DecisionBaseModel
 	{
 		private readonly ErrorService _errorService;
-		public DeclineReasonModel(ErrorService errorService, IAcademyConversionProjectRepository repository, ISession session) 
+
+		public DeclineReasonModel(ErrorService errorService, IAcademyConversionProjectRepository repository, ISession session)
 			: base(repository, session)
 		{
 			_errorService = errorService;
 			DeclineOptions = Enum.GetValues(typeof(AdvisoryBoardDeclinedReasons)).Cast<AdvisoryBoardDeclinedReasons>();
 		}
 
-		[BindProperty(Name = "DeclinedReasons"), MinLength(length: 1, ErrorMessage = "Select at least one reason")]
-		public IEnumerable<string> DeclinedReasons { get; set; }
+		[BindProperty] public IEnumerable<string> DeclinedReasons { get; set; }
```

Comment:
I managed to get the error message to be a link by only adding this to the property - (Name = "DeclinedReasons")